tests/acceptance/features/apiSharingNg/mountOrUnmountShare.feature

```diff
@@ -9,6 +9,9 @@ Feature:  mount or unmount incoming share
       | Alice    |
       | Brian    |
     And using spaces DAV path
+    And group "grp1" has been created
+    And user "Alice" has been added to group "grp1"
+    And user "Brian" has been added to group "grp1"
```

Comment:
```suggestion
    And group "grp1" has been created
    And user "Alice" has been added to group "grp1"
    And user "Brian" has been added to group "grp1"
```
I think it will be used in many of the scenario so no need to put in background